lint.sh

```diff
@@ -87,18 +88,23 @@ check_bash(){
   done < <(find ${fargs[@]} -iname \*.sh)
 }
 
-check_naming_standards() {
+# Check JJB for internal standards:
+#   * naming conventions
+#   * retention policy
+check_jjb_lint() {
   # Limits lint to only gating unique files
   # Excludes webhooktranslator as it has additional packaging
-  # and it's own tox
+  # and it's own tox.
   # Excludes NonCPS.groovy as this filename is required, but
   # not matching rpc-gating conventions
+  # Also runs other checks that require parsing JJB yaml including:
+  #   * Ensure retention policy is set for all jobs.
```

Comment:
These lines read like they were added before `check_naming_standards` was changed to `check_jjb_lint` and so should probably be removed.

Reply:
Done